app/scripts/templates/settings.mustache

```diff
@@ -1,24 +1,30 @@
 <div id="fxa-settings-header-wrapper">
   <header id="fxa-settings-header">
-    <h1 id="fxa-manage-account">{{#t}}Manage account{{/t}}</h1>
+    <h1 id="fxa-manage-account">{{#t}}Firefox Account{{/t}}</h1>
```

Comment:
Do you want `Account` or `Accounts` here. I understand the rationale for `Account`, but that doesn't seem right.